semapv/.htaccess

```diff
@@ -2,4 +2,4 @@ Options +FollowSymLinks
 RewriteEngine on
 
 RewriteRule ^$ https://github.com/mapping-commons/semantic-mapping-vocabulary [R=302]
-RewriteRule ^vocab/?$ https://mapping-commons.github.io/semantic-mapping-vocabulary/# [R=302]
+RewriteRule ^vocab\/(.*)$ https://mapping-commons.github.io/semantic-mapping-vocabulary/# [R=302]
```

Comment:
If you do ^vocab/(.*) you may get the result you are looking for. You don't need to escape slashes